### ISSUE TITLE
Center screen mode problem in gba

### ISSUE THREAD

**Issue by stangna** [Gericom/GBARunner2#215]
When playing any gba game in twilith menu, you always have 3 distorted stripes at the bottom of the game screen.

By deactivating the adjust mode and the central screen in the twilith menu options, this error disappears.

But when activating it gives this error.

Model: 3ds Old

**Comment by Epicpkmn11** [comment]
It's a TWL_FIRM bug made worse by GBARunner2 adjusting the framerate. You can either use an old build, more info [on the GBAtemp wiki](https://wiki.gbatemp.net/wiki/GBARunner2/DSi_3DS_Compatibility_List#TWLBg_Bug_-_Duplication_of_lines_on_the_bottom_of_the_screen_when_playing_on_a_.28N.292DS.2F3DS), or (the better option generally) use [VC Injects](https://3ds.eiphax.tech/nsui.html) or [open_agb_firm](https://github.com/profi200/open_agb_firm/releases).

GBARunner2 is primarily aimed at the DS and DSi, so on 3DS you have better options so you're really better off using those generally.

**Comment by michelemassa5599** [comment]
> It's a TWL_FIRM bug made worse by GBARunner2 adjusting the framerate. You can either use an old build, more info [on the GBAtemp wiki](https://wiki.gbatemp.net/wiki/GBARunner2/DSi_3DS_Compatibility_List#TWLBg_Bug_-_Duplication_of_lines_on_the_bottom_of_the_screen_when_playing_on_a_.28N.292DS.2F3DS), or (the better option generally) use [VC Injects](https://3ds.eiphax.tech/nsui.html) or [open_agb_firm](https://github.com/profi200/open_agb_firm/releases).
> 
> GBARunner2 is primarily aimed at the DS and DSi, so on 3DS you have better options so you're really better off using those generally.

No, i can't inject after installing cia file(s) in GodMode9 you will be:
```
0:/cia/Ace_lightning.cia
Error: MBR MBR is corrupt

(<A> to continue)
```
I pressed A button, then it appeared:
```
Exception: Data Abort (4)
GodMode9 v2.1.1
2022-03-31 11:30:09
 
R00: 00000001 | R01: 00000098
R02: FFFFFFFF | R03: FFEEFFEE
R04: 00000094 | R05: 00000000
R06: 00000000 | R07: 00213232
R08: 00000000 | R09: FFFFFFFF
R10: FFFFFFFF | R11: 08061E79
R12: 00000000 | R13: 228BFFB8
R14: 00000000 | R15: 08000543
CPSR: 200000D7

Press POWER to turn off.
```
And every time while i try to launch any game you will give me an error:
```
 An exception occurred
 
 Processor:       Arm9
 Exception type:  prefetch abort (svcBreak)
 
 R0     00000000       R1     00000000
 R2     08087965       R3     08088888
 R4     00000000       R5     00000000
 R6     00000000       R7     00000000
 R8     00000000       R9     FFFFFFFF
 R10    00000000       R11    00000000
 R12    00000000       SP     08FF0938
 LR     08028A8A       PC     0808580E
 CPSR   60000030
 
 You can find the dump in the following file:
 SD:/luma/dumps/arm9/crash_dump_00000001.dmp
 
 Press any button to shutdown
```
Meanwhile i tried to disable arm11 exception handlers, and the error screen is different:
```
 An error has occurred:
 
 Failed to apply 1 FIRM patch(es).
 
 Press any button to shutdown
```
Also i couldn't open open agb firm either:
```
 An error has occurred:
 
 The payload is invalid or corrupted.
 
 Press any button to shutdown
```
